addon/model-data.js

```diff
@@ -17,10 +18,38 @@ function commitDataAndNotify(modelData, updates) {
 class M3SchemaInterface {
   constructor(modelData) {
     this.modelData = modelData;
+    this._keyBeingResolved = null;
+    this._refKeyUnknownKeysMap = {};
```

Comment:
I'm slightly confused by the name here.

It's a dependency key map no?  why is it called `UnknownKeys`?

How about something like `_refKeyDepkeyMap`?

Or am I misunderstanding something?

Reply:
I will update, the key information.